source/administration/config-settings.rst

```diff
@@ -3964,8 +3975,20 @@ Enable Preview Features (Experimental)
 | This feature’s ``config.json`` setting is ``"EnablePreviewFeatures": true`` with options ``true`` and ``false`` for above settings respectively.                    |
 +---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Sidebar Organization (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: Enables channel sidebar organization options in **Account Settings** > **Sidebar** > **Channel grouping and sorting** including options for grouping unread channels, sorting channels by most recent post and combining all channel types into a single list. 
+
+**False**: Hides the channel sidebar organization options in **Account Settings** > **Sidebar** > **Channel grouping and sorting**.
+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature’s ``config.json`` setting is ``"ExperimentalChannelOrganization": false`` with options ``true`` and ``false`` for above settings respectively.         |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
 Group Unread Channels (Experimental)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*Removed in December 16, 2018 release*
```

Comment:
@esethna What happens to those with this setting enabled? Is the new setting then set to `true` by default for those systems?

Reply:
Do we also want to mention it was replaced by `ExperimentalChannelOrganization`, e.g. with something like

```
*Replaced by a new `ExperimentalChannelOrganization` setting in December 16, 2018 release*
```

or

```
*Removed in December 16, 2018 release and replaced by a new `ExperimentalChannelOrganization` setting*
```